src/map/battle.cpp

```diff
@@ -3903,6 +3905,16 @@ static void battle_calc_attack_masteries(struct Damage* wd, struct block_list *s
 #endif
 				}
 				break;
+#ifdef RENEWAL
+			case GN_CARTCANNON:
+				// These skills add arrow attack as mastery attack instead of equip attack
```

Comment:
```suggestion
				// This skill adds arrow attack as mastery attack instead of equip attack
```

Reply:
I put it in plural so people add similar skills to this list rather than creating their own checks.